### PR TITLE
1.19.21 Initial Commit

### PR DESCRIPTION
**Pull request by stevarino**
Looks like it is just the protocol version that changed, I can't find anything else and nothing is listed over at gophertunnel.

### CODE REVIEW

**Comment by stevarino** [comment]
Regarding the feedback in https://github.com/PrismarineJS/minecraft-data/pull/616 - unsure how to do a release PR but I think I got everything else.

File: doc/history.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 3.10.3
+* bedrock: add protocol data for 1.19.21 (@stevarino)
```
Comment (extremeheat) [inline doc/history.md:2]: History should be updated in a separate release PR so the GH release action can be triggered

**Comment by extremeheat** [comment]
A release PR is basically updating the HISTORY.md file and making a commit with the format of "Release x.x.x". The publish CI action will then be triggered. Should happen in a separate PR.

**Review by extremeheat — APPROVED** [review]

**Comment by extremeheat** [comment]
Thanks!

**Pull request merged**